testing/src/main/java/org/oppia/android/testing/profile/ProfileTestHelper.kt

```diff
@@ -103,16 +107,13 @@ class ProfileTestHelper @Inject constructor(
     return result
   }
 
-  /** ObserveForever is needed in order to complete the Action which returns the LiveData.
-   * Any Action which responds with the LiveData must need to observe it, to update the data
-   * on screen while running espresso test cases.
-   * */
+  /**
+   * While performing any action based on the LiveData, this helper function should be used.
+   * Without this helper function the LiveData cannot be accessed.
+   *
+   * @param data is the LiveData which needs to accessed while performing action.
```

Comment:
```suggestion
   * While performing any action based on the LiveData (see other methods above), this helper function should be used to ensure the operation corresponding to the LiveData properly completes.
   *
   * @param data is the LiveData which needs to accessed while performing action
```

Some suggested rewording (may require line-wrapping).

Reply:
Done